control-plane/pkg/reconciler/consumergroup/consumergroup_test.go

```diff
@@ -146,6 +149,7 @@ func TestReconcileKind(t *testing.T) {
 						}
 						_ = cg.MarkReconcileConsumersFailed("PropagateSubscriberURI", ErrNoSubscriberURI)
 						cg.MarkScheduleSucceeded()
+						cg.MarkAutoscalerDisabled() // KEDA not installed
```

Comment:
As a TODO can we also capture to add some unit tests with autoscaler enabled?

Reply:
Those tests have autoscaler feature `enabled` but KEDA is not installed in tests. Scaledobject lister etc fails. Will have to mock that. Adding as todo..